### ISSUE TITLE
[Feature request] nodejs caching

### ISSUE THREAD

**Issue by eviltik** [xenova/transformers.js#62]
Hi, thank you for your works

I'm a nodejs user and i read that there is no model cache implementation right now, and you are working on it.

Do you have an idea of when you will be able to push a release with a cache implementation ?

Just asking because i was at the point to code it on my side

**Comment by eviltik** [comment]
I'll not submit a PR because it's quick & dirty but here it is

https://github.com/eviltik/transformers.js/commit/bc624b93f070c980ec55583e8575a6238b5ed1f0

**Comment by xenova** [comment]
Hi! Yes, we do plan to add automatic model caching (like it is done in the python library). In the meantime, however, you can just download and store the model locally, then change some variables, like this:
```js
import { env } from "@xenova/transformers";

// Use local path for models.
env.remoteModels = false;
env.localURL = '/path/to/models/';

// (Optional) Set parent path of .wasm files. Defaults to use a CDN.
env.onnx.wasm.wasmPaths = '/path/to/files/';
```

**Comment by xenova** [comment]
Node caching has now been implemented in the latest version (2.0.0-alpha.0). See: https://www.npmjs.com/package/@xenova/transformers

By default, it will save to `./.cache`. but you can set your own as follows:
```js
import { env } from '@xenova/transformers';
env.cacheDir = '/path/to/cache/directory/';
```


For more information, see the documentation: https://huggingface.co/docs/transformers.js/api/env
